src/routes/session.ts

```diff
@@ -41,6 +41,11 @@ export const login = (onLogin: SetupOptions['onLogin']): RequestHandler => (
 				throw new NotFoundError('User not found.');
 			}
 
+			if (user.password == null) {
```

Comment:
@afitzek is this ok?  It'll presumably be detectable via timing

Reply:
Does `comparePassword` also work with `null`/`undefined` values? If so, then we can definitely drop this in favor of updating the typings, rather than copying the same error message here as well.